staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/apis/webhookadmission/v1/types.go

```diff
@@ -14,50 +14,51 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package encryptionconfig
+package v1
```

Comment:
why is this moved?

Reply:
Because the old type was removed as no one uses it anymore.

Reply:
And is this v1 quality? I doubt that. More like v1alpha1 with the many changes made.

Reply:
We have to stick with v1 because it's already documented and users already use it: https://github.com/kubernetes/kubernetes/pull/63032#issuecomment-383705778